### DISCUSSION TITLE
Episode 210: C++ Cryptic Crossword

### DISCUSSION THREAD

**Discussion by CharString** [codereport/adsp2#109] [Episode]
https://adspthepodcast.com/2024/11/29/Episode-210.html

**Comment by CharString** [comment]
The question is not "How many nerds that follow you know Wicket [W. Warwick] is an ewok?". The question is, "How many of them have the Kenner toy? And how many of those have the movie toy vs the tv series toy... And for extra nerd cred: have the movie toy with the box that doesn't have the name cut out..."

My parents used to be avid cryptogram (what we call them in Holland) solvers. And I amazed an English friend once, by solving the cryptic in his English newspaper. He ascribed me mythical proficiency in his native tongue, where it's mostly an associative and analytical mind.
What I love about well set cryptics, is that you just know correctness, because it's not merely about letters fitting or not; the whole construct fits. A bit like how the Curry-Howard-Lambek isomorphism gives us this solid confidence in results, because different fields point to the same construct.

This makes me think that current LLMs are bad at cryptic puzzles for other reasons than that it's impossible. There exists some vector space that encodes cryptic puzzles... Humanity just needs to nerdsnipe the right Scrabble™ aficionado with a knack for array programming and access to company hardware that can churn through lots of tensors...

**Comment by elbeno** [comment]
Agreed. I'll note here that cryptics are already well-served by various digital tools both on the setting and the solving side. As in many things IMO, the cost of using an LLM would be unjustified for the marginal improvement it may give.

And while I appreciate for a beginner that any form of getting to the answer is fair game, and that clear explanations (possibly computer-aided) are the way to learn, there comes a point where setting and solving cryptics is a distinctly human activity. For the same reason I wouldn't be very interested in reading a novel penned by AI, I wouldn't be very interested in solving a crossword set by AI. These things aren't only dry puzzles to be solved. They are a communication between setter and solver.

I'm just going to leave this here as an illustration of the human side of cryptics: https://www.youtube.com/watch?v=wk94NU5N0P8
RIP Araucaria.
